### PR TITLE
updating product to 1.2 license

### PR DESCRIPTION
**Pull request by mdubya66**

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,12 @@
+NOTICE
+
+vSphere Integrated Containers
+
+Copyright � 2017 VMware, Inc. All Rights Reserved.
```
Comment (anchal-agrawal) [inline NOTICE]: The copyright character looks off.

Comment (cgtexmex) [inline NOTICE]: (c) rendering as a _(?)_ -- feature or bug?

Comment (mdubya66) [inline NOTICE]: no idea. as it was delivered from legal

Comment (mhagen-vmware) [inline NOTICE]: that is just in github, if you click view on the file it renders correctly

**Pull request merged**